### DISCUSSION TITLE
How can I set the keys, token from twitter developer in out pc's environment variables?

### DISCUSSION THREAD

**Discussion by abbas2399** [MichaelCade/90DaysOfDevOps#261] [General]
I mean, the credentials we got from the Twitter developer account and on day 13 we were asked to set those credential variables in the environment variables of our personal OS.

I don't know what to do here and if done does it not disturb my os default settings?

**Comment by MichaelCade** [comment]
Sorry for the delay here, 

This blog from covers the areas I believe you are looking for, 

[How To Set Environment Variables](https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html)

I am also going to add this to the resources on Day 13 - 2022 edition.

**Comment by abbas2399** [comment]
Ok, I will look into it.
